components/Schedule/Schedule.tsx

```diff
@@ -26,54 +27,70 @@ export default function Schedule() {
   const [currentWeekOfTheYear, setCurrentWeekOfTheYear] = useState(getWeek())
   const [weekToDisplay, setWeekToDisplay] = useState(currentWeekOfTheYear)
 
+  useEffect(() => {
+    setCurrentWeekOfTheYear(getWeek())
+    console.log(currentWeekOfTheYear)
+  }, [])
+
   let { data, error, loading } = useQuery(ALL_CLASSES_QUERY)
   if (loading) return <p>Loading...</p>
   if (error) return <p>Error: {error.message}</p>
 
-  const changeWeek = (direction) => {
+  const changeWeek = (direction: string) => {
```

Comment:
You can also be more specific when you know `direction` is always a certain string.
So let's say direction always get's either "previous" or "next", you can type it like this:
```javascript
const changeWeek = (direction: 'previous' | 'next') => {
``` 
That way it breaks if any other string is passed, which makes your code safer

Reply:
Better yet:

Create an enum, maybe in your type files:
```
export enum Direction {
  PREVIOUS,
  NEXT,
}
```
 And use it in multiple places:
```javascript
const changeWeek = (direction: Direction) => {
    direction === Direction.PREVIOUS
```
And your button would be:
```javasscript
 <SmallButton
          type="button"
          onClick={() => changeWeek(DIRECTION.NEXT)}
          disabled={weekToDisplay > currentWeekOfTheYear + 1}
        >
          Next Week
        </SmallButton>

```